pkg/scheduler/framework/plugins/podtopologyspread/filtering_test.go

```diff
@@ -1424,6 +1452,7 @@ func TestMultipleConstraints(t *testing.T) {
 		nodes        []*v1.Node
 		existingPods []*v1.Pod
 		fits         map[string]bool
+		wantErrCode  map[string]framework.Code
```

Comment:
Now that I think of it, we only need `wantStatusCode map[string]framework.Code` to replace both `fits` and `wantErrCode`

Reply:
+1. Please help to update `TestSingleConstraint` as well.

Reply:
Done